qemu/tests/usb_host.py

```diff
@@ -60,6 +64,41 @@ def usb_dev_unplug(dev):
         if messages_del.find(match_del) == -1:
             test.fail("kernel didn't detect unplug")
 
+    def _get_usb_mount_point():
+        """ Get passthrough usb stick mount point """
+        cmd = "dmesg | grep 'Attached SCSI removable disk'"
+        s, o = session.cmd_status_output(cmd)
+        if s:
+            test.error("Fail to get passthrough usb stick in guest.")
+        dev = re.findall(r'\[(sd\w+)\]', o)[0]
+        cmd = "cat /proc/mounts | grep /dev/%s" % dev
+        s, o = session.cmd_status_output(cmd)
+        if s:
+            s = session.cmd_status('mount /dev/%s /mnt' % dev)
+            if s:
+                test.error("Fail to mount /dev/%s, output: %s" % (s, o))
```

Comment:
`(dev, o)`, and this `o` is not the output of mount action, it is the output L75.

several `cmd` do different things, better use different param name.

Reply:
Updated, thanks!